packages/blade/src/components/AutoComplete/_decisions/decisions.md

```diff
@@ -0,0 +1,253 @@
+
+```jsx
+<Dropdown selectionType="multiple">
+  <SelectInput label="Select Emails" />
+  <BottomSheet>
+    <BottomSheetHeader title="Do you want this report in an email?">
+      <AutoComplete label="Add receiver's email addresses" />
+    </BottomSheetHeader>
+    <BottomSheetBody>
+      <ActionList>
+        <ActionListItem title="Mumbai" value="mumbai" />
+        <ActionListItem title="Pune" value="pune" />
+        <ActionListItem title="Hyderabad" value="hyderabad" />
+      </ActionList>
+    </BottomSheetBody>
+  </BottomSheet>
+</Dropdown>
+```
+
+### With Tags Outside
+
+When possible selections are more than 10, it is advised to put tags outside of the Input.
+
+<img width="700" alt="image" src="https://github.com/razorpay/blade/assets/30949385/7ed7c93f-e434-4d29-ad10-222906810b7f">
+
+**Code**
+
+```jsx
+const [selectedItems, setSelectedItems] = React.useState([]);
+
+// ...
+
+<AutoComplete
+  label="Status"
+  labelPosition="inside-input"
+  maxRows="single"
+  value={selectedItems}
+  onChange={({ values }) => setSelectedItems(values)}
+/>
+
+
+// Somewhere outside in your UI
+<Box>
+  {selectedItems.map((selectedItemName, index) =>
+    <Tag
+      onDismiss={() => removeItem(index)}
+    >
+      {selectedItemName}
+    </Tag>
+  )}
+</Box>;
+```
+
+### With Different Label Positions
+
+While things are flawless when you give visible label (highly recommended), If there are scenarios where you are not able to give label to inputs (questionable but ok), you can use `labelPosition: 'inside-input'` prop.
+
+| **Code**                                                       | **Description**       | **Single Select**                                                                                                | **Multi Select**                                                                                                 |
+| -------------------------------------------------------------- | --------------------- | ---------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------- |
+| `<AutoComplete label="Status" /> `                             | With Label Present    | <img width="500" src="https://github.com/razorpay/blade/assets/30949385/0c559daa-5cdc-4d71-9221-6073b7969b1b" /> | <img width="500" src="https://github.com/razorpay/blade/assets/30949385/9314fd11-96b7-46ef-963f-283b393cb9fd" /> |
+| `<AutoComplete label="Status" labelPosition="inside-input" />` | Without Label Present | <img width="500" src="https://github.com/razorpay/blade/assets/30949385/2cb1d659-4a98-4f63-96d8-204ead4a681f" /> | <img width="500" src="https://github.com/razorpay/blade/assets/30949385/0b219c4c-2df2-4cfe-a32d-4c02809ab440" /> |
+
+### With Different Inactive - Active States
```

Comment:
Please add Positive/Negative intent examples

Reply:
yup this will be supported. This is basically `validationState` variable on input. 

I'll add a note about supporting common input properties in doc.